### PR TITLE
add SRC router preprocessor

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: ingest/processors/srcrouter.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+		}
+		if r.tag == `` {
+			r.drop = true
+		}
+		rts = append(rts, r)
+	}
+	return
+}
+
+func getSrcRoute(v string) (a string, b string, err error) {
```
Comment (traetox) [inline ingest/processors/srcrouter.go:166]: can we have this execute an `ingest.CheckTag(b)` here too, that way we catch bag tags right out of the gate.

File: ingest/processors/srcrouter_test.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+const ()
+
+var (
+	src = SrcRouteConfig{
+		Route: []string{`1.1.1.1:foo`, `2.2.2.2:bar`, `3.3.3.3:`},
+	}
+)
+
+func TestSrcRouteConfig(t *testing.T) {
```
Comment (traetox) [inline ingest/processors/srcrouter_test.go:27]: add in a test with some bad route defs and some files that are malformed?

**Review by traetox — CHANGES_REQUESTED** [review]

**Review by djfritz — COMMENTED** [review]
Agree with @traetox's comments, otherwise LGTM

**Comment by floren** [comment]
PTAL

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**